website/content/docs/connect/failover/sameness.mdx

```diff
@@ -0,0 +1,206 @@
+
+You can establish a failover strategy by configuring sameness group behavior in the following locations:
+
+- Sameness group configuration entry
+- Service resolver configuration entry
+- Prepared queries
+
+### Failover with a sameness group configuration entry
+
+To define failover behavior using a sameness group configuration entry, set `DefaultForFailover=true` and then apply the updated configuration to all clusters that are members of the group.
+
+When `DefaultForFailover=true`, Consul attempts to redirect traffic to healthy instances according to the order of the group `Members` in the configuration entry. When you set `IncludeLocal=true` in the sameness group configuration entry, Consul attempts to redirect traffic to another instance in the same admin partition before redirecting to other group members.
+
+In the following example configuration entry, datacenter `dc1` has two partitions, `partition-1` and `partition-2`. A second datacenter, `dc2`, has a single partition named `partition-1`. All three partitions have identically configured services and established cluster peering connections. The configuration entry defines a sameness group, `example-sg` in `dc1`. When redirecting traffic during a failover scenario, Consul will attempt to find a healthy instance in a specific order: `dc1-partition-1`, then `dc-1-partition-2`, then `dc2-partition-1`.
+
+<Tabs>
+
+<Tab heading="HCL" group="hcl">
+
+```hcl
+Kind                     = "sameness-group"  
+Name                     = "example-sg"
+Partition                = "partition-1" 
+DefaultForFailover       = true
+IncludeLocal             = true
+Members                = [
+      {Partition = "partition-1"},
+      {Partition = "partition-2"},
+      {Peer      = "dc2-partition-1"}
+      ]
+```
+
+</Tab>
+
+<Tab heading="JSON" group="json">
+
+```
+{
+    "Kind": "sameness-group", 
+    "Name": "example-sg", 
+    "Partition": "partition-1",
+    "DefaultForFailover": true,
+    "IncludeLocal": true,
+    "Members": [
+        {
+            "Partition": "partition-1"
+        },
+        {
+            "Partition": "partition-2"
+        },
+        {
+            "Peer": "dc2-partition-1"
+        }
+    ]
+}
+```
+
+</Tab>
+
+<Tab heading="YAML" group="yaml">
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: SamenessGroup 
+metadata:
+  name: example-sg
+spec:
+  defaultForFailover: true
+  includeLocal: true
+  members:                                   
+    - partition: partition-1
+    - partition: partition-2
+    - peer: dc2-partition-1
+```
+
+</Tab>
+</Tabs>
+
+When a sameness group is configured as the failover default, sameness group failover takes precedence over all other failover configurations in the cluster.
```

Comment:
This is not true. The `defaultForFailover=true` makes it so that the SG is used for failovers whenever users have not defined their own failover in a service-resolver for a service. Whenever a resolver is defined for an upstream, it will be used instead of the SG defaultForFailover behavior.